### ISSUE TITLE
[regression] error: 'module' object has no attribute 'ReferenceNotFoundException'

### ISSUE THREAD

**Issue by yurivict** [pubs/pubs#165]
Version ```0.8.0``` fails:

```
$ pubs add -d ~/Downloads/x.pdf -D {some-DOI}
error: 'module' object has no attribute 'ReferenceNotFoundException'
```

```pubs-0.7.0.104``` didn't have this problem.

**Comment by benureau** [comment]
Thanks for this. I forgot to properly test a function. I'll push a fix ASAP.

**Comment by benureau** [comment]
@yurivict: I pushed a fix in the `fix/doi_stats` branch. It will be merged soon into master and we'll publish a hotfix release (0.8.1) right after that. In the meantime, you can install the updated version with:
```
pip install -U git+https://github.com/pubs/pubs@fix/doi_stats
```

**Comment by yurivict** [comment]
Thank you!

**Comment by benureau** [comment]
Included in version 0.8.1. Closed by 8fe24140.

**Comment by yurivict** [comment]
Thank you for fixing it!
I updated the port.
